test/e2e/grpc_test.go

```diff
@@ -78,10 +78,14 @@ func unaryTest(t *testing.T, resources *v1a1test.ResourceObjects, clients *test.
 	t.Helper()
 	t.Logf("Connecting to grpc-ping using host %q and authority %q", host, domain)
 	conn, err := dial(host, domain)
+	defer func() {
+		if conn != nil {
+			conn.Close()
+		}
+	}()
```

Comment:
Does this change actually do anything? I looked through `grpc`'s `Dial` function and it seems to always return `nil` in case of an error anyway. It also has it's own defer to tear down connections in case of an error.

Reply:
You are right. I have recovered it.